articles/static-web-apps/overview.md

```diff
@@ -40,7 +40,7 @@ With Static Web Apps, static assets are separated from a traditional web server
 
 ## What you can do with Static Web Apps
 
-- **Build modern JavaScript applications** with frameworks and libraries like [Angular](https://angular.io/), [React](https://reactjs.org/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/) with an [Azure Functions](https://azure.microsoft.com/services/functions/) back-end.
+- **Build modern JavaScript applications** with frameworks and libraries like [Angular](getting-started.md#tab/angular), [React](getting-started.md#tab/react), [Svelte](https://docs.microsoft.com/en-us/learn/modules/publish-app-service-static-web-app-api/), [Vue](getting-started.md#tab/vue) with an [Azure Functions](apis.md) back-end.
```

Comment:
```suggestion
- **Build modern JavaScript applications** with frameworks and libraries like [Angular](getting-started.md#tabs=angular), [React](getting-started.md#tabs=react), [Svelte](https://docs.microsoft.com/learn/modules/publish-app-service-static-web-app-api/), [Vue](getting-started.md#tabs=vue) with an [Azure Functions](apis.md) back-end.
```
This fixes the document fragment key and removes the locale from the Learn URL.

Reply:
@craigshoemaker thanks for the review! Changes applied